invisible_cities/cities/components_test.py

```diff
@@ -149,15 +151,15 @@ def dummy_city(files_in, file_out, event_range):
 
 def test_hits_and_kdst_from_files(ICDATADIR):
     event_number = 1
-    timestamp    = 0.
+    # timestamp    = 0.
```

Comment:
This test is unrelated so shouldn't be changed

Reply:
Okay, I had assumed that this test would fail and I did not get to check it.